packages/components/src/List/ListToVirtualizedList/ListToVirtualizedList.component.js

```diff
@@ -51,6 +52,12 @@ export function ListToVirtualizedList(props) {
 				supposedActions[key] = true;
 			});
 	}
+
+	// allow to connect the CellTitle
+	const InjectedCellTitle = Inject.get(props.getComponent, 'CellTitle', CellTitle.cellRenderer);
```

Comment:
This doesn't scale.
If a project need to connect another cellRenderer, to inject a new one, or to replace an existing one, thay will have to add it here :/

In fact I just saw that we already have the system for that.
* List accept cellDictionary props
* ListToVirtualizedList merge the dictionaries `const listCellDictionary = { ...cellDictionary, ...props.cellDictionary };`

So what needs to be done : 
* Containers connect the CellTitle and export it (instead of exporting it in Components)
* List container Inject the connected cellTitle and pass it in the props.cellDictionary to the List component
* ListToVirtualizedList must use `listCellDictionary['title']` instead of `CellTitle` directly. It doesn't need the inject or a getComponent as props anymore.